### PR TITLE
Task-46050: Implement news analytics board

### PR DESCRIPTION
**Pull request by AzmiTouil**
Prior to this change, we are not able to calculate metrics for news operations. This PR will implement news analytics board

### CODE REVIEW

File: services/src/main/java/org/exoplatform/news/listener/NewsGamificationIntegrationListener.java [hunk 1]
```diff
@@ -57,8 +51,8 @@ public void onEvent(Event<String, String> event) throws Exception {
         Map<String, String> gamificationMap = new HashMap<>();
         gamificationMap.put("ruleTitle", ruleTitle);
         gamificationMap.put("object", news.getUrl());
-        gamificationMap.put("senderId", earnerId); // matches the gamification's earner id
-        gamificationMap.put("receiverId", earnerId);
+        gamificationMap.put("senderId", news.getAuthor()); // matches the gamification's earner id
+        gamificationMap.put("receiverId", news.getAuthor());
```
Comment (ahamdi) [inline services/src/main/java/org/exoplatform/news/listener/NewsGamificationIntegrationListener.java:55]: I think we need just the senderId , no need to store the receiverId right ?

File: services/src/test/java/org/exoplatform/news/rest/NewsRestResourcesV1Test.java [hunk 2]
```diff
@@ -1954,4 +1955,62 @@ public void shouldGetStagedNewsWhenCurrentUserIsAuthor() throws Exception {
     assertEquals("john", newsList.get(0).getAuthor());
   }
 
+  @Test
+  public void getNewsIllustrationTest() {
```
Comment (ahamdi) [inline services/src/test/java/org/exoplatform/news/rest/NewsRestResourcesV1Test.java:2062]: UT are not related to the added code. I understand that you want to keep the same test coverage. That is fine

File: services/src/main/java/org/exoplatform/news/listener/NewsGamificationIntegrationListener.java (hunk 1 above)
Comment (AzmiTouil) [reply services/src/main/java/org/exoplatform/news/listener/NewsGamificationIntegrationListener.java:55]: The gamification object map must contain both the senderId and receiverId params.

**Review by ahamdi — DISMISSED** [review]

**Review by ahamdi — DISMISSED** [review]

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**